account_banking_sepa_direct_debit/data/mandate_expire_cron.xml

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8" ?>
+<?xml version="1.0" ?>
```

Comment:
Please @JasminSForgeFlow revert this kind of change in the .xml files. The warning is for .html files.

Reply:
Updated, Thanks